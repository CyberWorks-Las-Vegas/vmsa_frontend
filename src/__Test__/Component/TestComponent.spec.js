import React from "react";
import { create } from "react-test-renderer";

import BoshokaComponent from "../../Components/BoshokaComponent";

describe("Boshoka component", () => {
  test("Matches the snapshot", () => {
    const boshoka = create(
      <BoshokaComponent props={{ msg: "Hello World!", repeat: 5 }} />
    );
    expect(boshoka.toJSON()).toMatchSnapshot();
  });
});
