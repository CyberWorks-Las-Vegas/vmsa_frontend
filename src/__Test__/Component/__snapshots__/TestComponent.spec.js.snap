// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Boshoka component Matches the snapshot 1`] = `
<ul>
  <li>
    <h1>
      Hello World!
    </h1>
  </li>
  <br />
  <li>
    <h1>
      Hello World!
    </h1>
  </li>
  <br />
  <li>
    <h1>
      Hello World!
    </h1>
  </li>
  <br />
  <li>
    <h1>
      Hello World!
    </h1>
  </li>
  <br />
  <li>
    <h1>
      Hello World!
    </h1>
  </li>
  <br />
</ul>
`;
